terraform/ccnew/default-config/cluster-config.yaml

```diff
@@ -25,9 +26,10 @@ nodes:
     node_taints: []
     node_labels: []
     extra_vol: true
-    extra_vols:
-      - name: "/dev/sdh"
-        size: 100
+    extra_vols: []
```

Comment:
we should show an example here on how to configure the storage per node pool

Reply:
Please share an example if possible I didn't get your point well.

Reply:
how do you configure the storage per different node pool configutations?   @sijo5722-2021 would you chime in here?